operator/gitops/argocd/pipeline-service/tekton-results/base/kustomization.yaml

```diff
@@ -50,10 +50,20 @@ patchesJson6902:
           - "/bin/bash"
           - "-c"
           - |
-            if oc get secret -n tekton-results tekton-results-database 2>&1 >/dev/null; then exit; fi
-            oc create secret generic tekton-results-database \
+            if oc get secret -n tekton-results tekton-results-postgres 2>&1 >/dev/null; then
+              exit
+            fi
+            if oc get secret -n tekton-results tekton-results-database 2>&1 >/dev/null; then
+              DATABASE_USER=$(oc get secret -n tekton-results tekton-results-database -o jsonpath="{.data.DATABASE_USER}")
+              DATABASE_PASSWORD=$(oc get secret -n tekton-results tekton-results-database -o jsonpath="{.data.DATABASE_PASSWORD}")
+              oc delete secret -n tekton-results tekton-results-database
+            else
+              DATABASE_USER="tekton"
+              DATABASE_PASSWORD="$(dd if=/dev/urandom bs=20 count=1 2>/dev/null | base64)"
```

Comment:
Might wanna change `DATABASE_...` to `POSTGRES_...` as well, not important but will be more clear naming IMO.

Reply:
Fixed